dev/com.ibm.ws.kernel.boot.nested/src/com/ibm/ws/kernel/launch/internal/LauncherDelegateImpl.java

```diff
@@ -139,6 +139,24 @@ public boolean shutdown() throws InterruptedException {
         return true;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @throws InterruptedException
+     */
+    @Override
+    public boolean shutdown(boolean force) throws InterruptedException {
```

Comment:
Same comment here about having the no-arg version call this version with `false`.